src/nvme/mi-mctp.c

```diff
@@ -459,12 +638,15 @@ static const struct nvme_mi_transport nvme_mi_transport_mctp = {
 	.submit = nvme_mi_mctp_submit,
 	.close = nvme_mi_mctp_close,
 	.desc_ep = nvme_mi_mctp_desc_ep,
+	.async_read = nvme_mi_mctp_read,
+	.async_poll_fd = nvme_mi_mctp_async_poll_fd,
```

Comment:
It's only a minor naming convention, but we probably want to be careful between the terms `async` (as an IO-handling pattern) and "Async Event Messaging" (as the NVMe-MI facility). I would suggest using `aem` consistently here.